visualization/app/codeCharta/codeCharta.model.ts

```diff
@@ -141,6 +141,7 @@ export interface AppSettings {
 	mapColors: MapColors
 	isPresentationMode: boolean
 	showOnlyBuildingsWithEdges: boolean
+	edgeMetricToggler: boolean
```

Comment:
I would change the naming to not use a noun because it is a simple boolean flag.

Reply:
You're right. I renamed everything for a better understanding.